yarn.lock

```diff
@@ -2,6 +2,13 @@
 # yarn lockfile v1
```

Comment:
can you use `npm ci ` for installing the packages. then you'll have the correct version based on the lockfile

Reply:
I think you still need to remove the yarn.lock file, we don't want to push it to the main repo